core-java/src/main/java/com/baeldung/ZipCollectionExample.java

```diff
@@ -0,0 +1,41 @@
+package com.baeldung.guava.tutorial;
+
+import com.google.common.collect.Streams;
+import org.jooq.lambda.Seq;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class ZipCollectionExample {
+    static List<String> names = new ArrayList<>(Arrays.asList("John", "Jane", "Jack", "Dennis"));
```

Comment:
Is there any reason why we can't simply use Arrays.asList without passing that to ArrayList?